### PR TITLE
Demo/login with sms

### PR DESCRIPTION
**Pull request by HilaPartukTS**

### CODE REVIEW

File: login-with-sms/pages/sms-otp.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      phone: phone,
+    }),
+  })
+
+  const status = response.status
+  const data = await response.json()
+  console.log('Response from /email-otp', { status, data })
```
Comment (ts-ron) [inline login-with-sms/pages/sms-otp.js:14]: fix text: email to sms

**Review by ts-ron — COMMENTED** [review]
Overall LGTM - see commments mostly around leftover text referring to email instead of SMS

File: login-with-sms/backend/index.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+// For more information see https://developer.transmitsecurity.com/guides/user/auth_email_otp/#step-5-obtain-user-token
+router.get('/complete', function (req, res) {
+  if (req.query.code) {
+    res.send(`Login completed with code: ${escape(req.query.code)}`)
+  } else {
+    res.send(`Login completed with error: ${escape(req.query.error)}`)
+  }
+})
+
+// For more information see https://developer.transmitsecurity.com/guides/user/auth_email_otp/#step-3-send-email-otp
```
[the remark this answers is not in the document]
Comment (HilaPartukTS) [reply login-with-sms/backend/index.js]: fixed all email references

File: login-with-sms/pages/sms-otp.js (hunk 1 above)
Comment (HilaPartukTS) [reply login-with-sms/pages/sms-otp.js]: fixed all email references

File: login-with-sms/backend/index.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+let accessToken = null
+
+// GET login page
+router.get('/', function (req, res) {
+  res.redirect('/pages/sms-otp.html')
+})
+
+// The following endpoint is used by pages/sms-otp.html during a login flow
+// It uses an API to send and email with the OTP code to the user
+// For more information see https://developer.transmitsecurity.com/guides/user/auth_email_otp/#step-3-send-email-otp
```
[the remark this answers is not in the document]
Comment (HilaPartukTS) [reply login-with-sms/backend/index.js]: fixed all email references

File: login-with-sms/backend/index.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+// This parameter emulates this 'cache' with a static variable for simplicity.
+let accessToken = null
+
+// GET login page
+router.get('/', function (req, res) {
+  res.redirect('/pages/sms-otp.html')
+})
+
+// The following endpoint is used by pages/sms-otp.html during a login flow
+// It uses an API to send and email with the OTP code to the user
```
[the remark this answers is not in the document]
Comment (HilaPartukTS) [reply login-with-sms/backend/index.js]: fixed all email references

**Review by ts-ron — APPROVED** [review]
LGTM 🆒 ❄️

**Pull request merged**